iosApp/iosApp/Pages/StopDetails/StopDetailsPageHandlerExtension.swift

```diff
@@ -0,0 +1,183 @@
+//
+//  StopDetailsPageHandlerExtension.swift
+//  iosApp
+//
+//  Created by esimon on 11/20/24.
+//  Copyright © 2024 MBTA. All rights reserved.
+//
+
+import shared
+import SwiftPhoenixClient
+import SwiftUI
+
+extension StopDetailsPage {
```

Comment:
suggestion (non-blocking): I like this split out! Since this is all logic-focused, I think it could be worth moving it into a ViewModel. I think that would help with testability - you could test the ViewModel directly, and mock ViewModel function calls where helpful to unit test the view.

Reply:
Yeah that's a good idea! I can add that as an AC on the next ticket.